packages/frontend-shared/src/gql-components/topnav/UpdateCypressModal.vue

```diff
@@ -4,6 +4,7 @@
     variant="bare"
     :title="title"
     :model-value="show"
+    help-link=""
```

Comment:
I am not really comfortable with this.
An empty sting does not mean "hide the link" in my book. It means show the link with an empty target.

I would prefer a quick refactoring of the modal to make is so to this empty prop.

Reply:
If you're comfortable _reviewing_ that change, I'm happy to make it. I thought about that but a) it's probably not my place to make that decision and we already had this pattern and were using it and b) I was worried I might break something and not see it. It's very hard to find all these modal states in the app or in the code to confirm that they all have the correct behavior.

Reply:
Can I suggest we create a ticket to refactor the modal help links? 
This way the technical debt accumulated here is tracked somewhere.
As soon I see the ticket, it is green to me.

Reply:
[Done](https://cypress-io.atlassian.net/browse/UNIFY-1197)!